tests/ruling/snapshots/no-identical-expressions

```diff
@@ -3,6 +3,7 @@ src/angular/packages/core/src/view/types.ts: 287
 src/angular/packages/benchpress/src/metric/perflog_metric.ts: 319
 src/angular/packages/compiler/test/aot/static_reflector_spec.ts: 187,194,201,208,215,224,233,242
 src/angular/tools/@angular/tsc-wrapped/test/evaluator.spec.ts: 101
+src/desktop/app/test/unit/progress/clone-test.ts: 42
```

Comment:
Maybe this issue is new because this PR does not use the same commit for typescript-test-sources as the master.